### PR TITLE
Add mandatory listing date tot market config in order to hide APY on …

### PR DESCRIPTION
**Pull request by midas-myth**
…fresh markets

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207859482138074

### CODE REVIEW

File: src/config/markets.ts [hunk 1]
```diff
@@ -1,58 +1,129 @@
...
 export function isMarketEnabled(chainId: number, marketAddress: string) {
   if (isDevelopment()) return true;
 
   return ENABLED_MARKETS_INDEX[chainId]?.[marketAddress] ?? false;
 }
+
+/**
+ * We let the APY to stabilize for a few days before showing it to the user
+ */
+export function getFutureDateForApyReadiness(chainId: number, marketAddress: string): undefined | Date {
```
Comment (gmxer) [inline src/config/markets.ts:117]: it's in fact not a future date, is it?  it's just a start (listing) date

maybe `getMarketListingDate`, `getMarketApyStartDate` or `getMarketApyReadinessDate` instead?

File: src/config/markets.ts [hunk 2]
```diff
@@ -1,58 +1,129 @@
...
+  const tokenListing = ENABLED_MARKETS[chainId]?.[marketAddress];
+
+  if (!tokenListing) return undefined;
+
+  const enoughDateForApy = addDays(tokenListing.listingDate, ENOUGH_DAYS_FOR_APY);
```
Comment (gmxer) [inline src/config/markets.ts:122]: not sure if we should move this logic to AprInfo or even separate utililty file?
cos `config/tokens` provides fairly straightforward getters/setters whereas here it clamps dates

**Review by gmxer — CHANGES_REQUESTED** [review]
I see only deletions in translations files, maybe you missed pushing translations?

File: src/config/markets.ts (hunk 2 above)
Comment (midas-myth) [reply src/config/markets.ts:122]: but its basically a getter

File: src/config/markets.ts (hunk 1 above)
Comment (midas-myth) [reply src/config/markets.ts:117]: No, the function returns a date only if its in the future

**Comment by midas-myth** [comment]
> I see only deletions in translations files, maybe you missed pushing translations?

No, i did not, the diff shows a lot of deletions of comments. but there are my additions in there too

File: src/config/markets.ts (hunk 1 above)
Comment (midas-myth) [reply src/config/markets.ts:117]: if you want we can rename this function

File: src/config/markets.ts (hunk 2 above)
Comment (midas-myth) [reply src/config/markets.ts:122]: i could move it tot aprInfo but i would need to pass token address to apy unfo. if you think that is ok let me know.

File: src/components/AprInfo/AprInfo.tsx [hunk 3]
```diff
@@ -1,36 +1,58 @@
...
-  /**
-   * @default undefined meaning that APY is ready to be shown
-   */
-  futureDateForApyReadiness?: Date;
+  tokenAddress: string;
 }) {
   const { chainId } = useChainId();
+
+  const listingDate = getMarketListingDate(chainId, tokenAddress);
+  const isBaseAprReadyToBeShown = getIsBaseApyReadyToBeShown(listingDate);
```
Comment (gmxer) [inline src/components/AprInfo/AprInfo.tsx:45]: lets move this line inside useMemo below? you anyway store `Date | undefined` in `apyReadyToBeShownDate` var

**Review by gmxer — CHANGES_REQUESTED** [review]
one small comment, besides that looks nice 👍

File: src/components/AprInfo/AprInfo.tsx (hunk 3 above)
Comment (midas-myth) [reply src/components/AprInfo/AprInfo.tsx]: fixed

**Review by gmxer — APPROVED** [review]

**Pull request merged**